reactor-core/src/test/java/reactor/core/scheduler/GenericThreadPerTaskBoundedElasticSchedulerTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023 VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.core.scheduler;
+
+import org.assertj.core.api.Assumptions;
+import org.junit.jupiter.api.Assertions;
+
+class GenericThreadPerTaskBoundedElasticSchedulerTest extends AbstractSchedulerTest {
+
+
+	static boolean SUPPORTED;
+
+	static {
+		try {
+			new VirtualThreadFactory("threadPerTaskBoundedElasticSchedulerTest",
+					false,
+					Schedulers::defaultUncaughtException);
+			SUPPORTED = true;
+		}
+		catch (Throwable t) {
+			SUPPORTED = false;
+		}
+	}
+
+	@Override
+	protected Scheduler scheduler() {
+		Assumptions.assumeThat(SUPPORTED).isTrue();
+		ThreadPerTaskBoundedElasticScheduler test =
```

Comment:
The `freshScheduler()` method could be called here.

Reply:
no need to do so. Just following the overall pattern that is makes no sense

Reply:
fixed